cypress/integration/navigation.spec.js

```diff
@@ -0,0 +1,38 @@
+context("Navigation", () => {
+  beforeEach(() => {
+    cy.visitPage("navigation");
+  });
+
+  it("displays the navigation items at large sizes", () => {
+    cy.findByRole("link", { name: "Products" }).should("be.visible");
+    cy.findByRole("button", { name: "Menu" }).should("not.exist");
+  });
+
+  it("displays the mobile menu at small sizes", () => {
+    cy.viewport('iphone-6')
+    cy.findByRole("link", { name: "Products" }).should("not.exist");
+    cy.findByRole("button", { name: "Menu" }).should("be.visible");
+  });
+
+  it("can open the mobile menu", () => {
+    cy.viewport('iphone-6')
+    cy.findByRole("link", { name: "Products" }).should("not.exist");
+    cy.findByRole("button", { name: "Menu" }).click(0, 0);
```

Comment:
I'm assuming this didn't work without passing exact pixels to click within the button and it's to workaround some visibility issue?

Reply:
Oh no, I just referenced how it was done in some of the other tests, but I'll remove that.